google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -2219,6 +2220,27 @@ public void testDmlStatistics() throws InterruptedException {
     assertEquals(2L, statistics.getDmlStats().getUpdatedRowCount().longValue());
   }
 
+  @Test
+  public void testTransactionInfo() throws InterruptedException {
+    String tableName = TABLE_ID_FASTQUERY.getTable();
```

Comment:
Do you need to worry about test ordering, or is this created in a setup?

Reply:
Thank you for pointing this out. Yes, this is fine since the later test case will still update the table given the `WHERE TRUE` condition.